dev-python/pymupdf/pymupdf-1.19.6.recipe

```diff
@@ -25,13 +25,11 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libmupdf$secondaryArchSuffix
-	lib:libfreetype$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libmupdf$secondaryArchSuffix
+	devel:libmupdf$secondaryArchSuffix >= ${portVersion%.**}
```

Comment:
As mentioned, don't use $portVersion here:
```diff
- 	devel:libmupdf$secondaryArchSuffix >= ${portVersion%.**}
+ 	devel:libmupdf$secondaryArchSuffix >= 1.19
```

Reply:
of course plain `$portVersion` wouldn't work as it could have a different patch revision but that is why I'm using the substitution

Reply:
If you check on other recipes (eg boost related), you would see none of them use $portVersion variables there, it's common to use the depending version (even if in this case it's the same) (PS, didn't check this change), but what if pymupdf bumpt to 1.20.* before mupdf does?

Reply:
In this case I am quite sure that won't happen - it would be meaningless to for the maintainers to release pymupdf 1.20.* without mupdf 1.20.*
Anyway I am happy to conform to the convention used by other ports so I'll push the change in a moment :)

Reply:
> it's common to use the depending version (even if in this case it's the same)
My point here is that the depending version is **absolutely always** the same between pymupdf and mupdf - pymupdf 1.19.x won't work with mupdf 1.18.x or 1.20.x or any other version of mupdf except for 1.19.x, because mupdf breaks compatibility with every minor version release 
That's why it seems logical to me to do this, but I'm happy to go with whatever you think is best and get these ports merged :)

Reply:
ATM pymupdf is already at 1.19.6 while mupdf is still at 1.19.1, I can't tell what the developers agree there, but better safe then sorry, dependency should rely on requires, not on the own version (now I'm closing down for today) :)

Reply:
Cheers, have a good evening

Reply:
pymupdf uses the static library of mupdf on Arch Linux, this is surely another "good" option  ...

Reply:
> pymupdf uses the static library of mupdf on Arch Linux, this is surely another "good" option ...

Yes I suppose it is a decent option - the unstable API is the reason MuPDF didn't support building as a shared library until recently. I tend to prefer the shared library though as it is more efficient and MuPDF is quite big.

Reply:
Sure, depends on the number of users, and how often it is updated.

Reply:
Well you could provide both shared and static library for mupdf, not sure if we want to go there, so far from my checks pymupdf seems ok with the shared library.

Reply:
Yeah I don't really want to go there. I am happy to keep MuPDF and pymupdf updated together and if anyone uses it and an issue arises look into providing a static lib as well.